### PR TITLE
USHIFT-1313 Test: Rebooting unhealthy system results in restoring data from a backup

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
Depends (and current includes) on https://github.com/openshift/microshift/pull/1866

File: test/suites-ostree/upgrade-ostree.robot [hunk 1]
```diff
@@ -76,3 +93,49 @@ System Should Be Healthy
...
+
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    microshift admin data backup --name "${backup_name}"
+    ...    sudo=True    return_stderr=True    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+
+    RETURN    ${backup_name}
+
+Mark System As Unhealthy
+    [Documentation]    Marks systems as unhealthy by executing greeboot's red script
```
Comment (dhellmann) [inline test/suites-ostree/upgrade-ostree.robot:113]:
```suggestion
    [Documentation]    Marks systems as unhealthy by executing microshift's red script
```

File: test/suites-ostree/upgrade-ostree.robot [hunk 2]
```diff
@@ -76,3 +93,49 @@ System Should Be Healthy
...
+    [Documentation]    Creates a marker file in backup directory
+    [Arguments]    ${backup_name}
+
+    # create a marker that we expect to show up in data directory after restore
+    ${stdout}    ${stderr}    ${rc}=    Execute Command
+    ...    touch ${BACKUP_STORAGE}/${backup_name}/marker
+    ...    sudo=True    return_stderr=True    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+
+Check If Marker Exists In Data Dir
```
Comment (dhellmann) [inline test/suites-ostree/upgrade-ostree.robot:129]: nit: naming suggestion to match other similar keywords

```suggestion
Marker Should Exist In Data Dir
```

**Review by dhellmann — COMMENTED** [review]
/test images
/lgtm

This looks good. I had a couple of nitpicky suggestions, but I'll leave it up to you to decide if you want to make the changes.

/hold
@jogeo do you want to assign QE reviewer before we merge it?

**Pull request merged**